Assets/Scripts/InventoryObject.cs

```diff
@@ -4,12 +4,37 @@
 
 public class InventoryObject : InteractiveObject
 {
-    // TODO: Add long description field
-    // TODO: Add icon field
+    [Tooltip("The name of the object, as it will appear in the inventory menu UI.")]
```

Comment:
Again, probably should put all the SerializeFields into a region, that goes for just about any script where you have multiple. (I think I forgot to do this myself, so it's a good reminder for me too)